pyxl/codec/fast_register.py

```diff
@@ -0,0 +1,9 @@
+import codecs
+
+def search_function(encoding):
+    if encoding != 'pyxl':
+        return None
+    import pyxl.codec.register
```

Comment:
This import costs 200-300 nsec on my machine. Not much, but still... A None check only costs 20-30 nsec. Maybe it's worth creating a global variable to hold the imported module, initialized to None, and only doing the import when it's still None?

Reply:
Per our discussion, we probably don't care?